### PR TITLE
Create cody-gitlab-integration blog post

### PR DESCRIPTION
**Pull request by amieroth**
GitLab partnership announcement blog post

### CODE REVIEW

File: content/blogposts/2023/cody-gitlab-integration [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Cody Now Integrates with GitLab
+description: GitLab customers can leverage Cody to tap into the full knowledge of their codebase, no matter the size, with a simple chat interface
+publishDate: 2023-06-26T12:00
+tags: [blog]
+slug: cody-gitlab-integration
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/Sourcegraph%20x%20Gitlab.png
+socialImage: Use to set large social image i.e.  https://storage.googleapis.com/sourcegraph-assets/blog/Sourcegraph%20x%20Gitlab.png
```
Comment (iskyOS) [inline content/blogposts/2023/cody-gitlab-integration:9]: @amieroth I think this is the issue
```suggestion
socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/Sourcegraph%20x%20Gitlab.png
```

File: content/blogposts/2023/cody-gitlab-integration [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+Today, Sourcegraph announced that Cody now integrates with GitLab, the most comprehensive AI-powered enterprise DevSecOps platform. Cody provides customers with multiple ways to take advantage of AI capabilities. 
+
+Key features of the integration include:
+* **Chat-based interface in your IDE:** Cody natively integrates with VS Code and comes pre-packaged with 9 recipes that allow devs to explain, understand, generate, translate, summarize, and detect ‘code smell’ across repositories and even generate fixes and unit tests. 
+* **Powerful and accurate AI:** Cody leverages Sourcegraph’s code graph to fetch code context, resulting in more precise outputs in line with an organization’s codebase and best practices. Developers can work more efficiently and intelligently within their editors while referencing knowledge from their entire codebase via AI.
+* **Accelerated Development:** By seamlessly connecting an organization's private codebase, documentation, and language libraries, Cody empowers developers to move even faster, delivering results quickly and reliably.
```
Comment (sqs) [inline content/blogposts/2023/cody-gitlab-integration:22]:
```suggestion
* **Accelerated development:** By seamlessly connecting an organization's private codebase, documentation, and language libraries, Cody empowers developers to move even faster, delivering results quickly and reliably.
```

File: content/blogposts/2023/cody-gitlab-integration [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+With Sourcegraph’s Cody, customers leveraging the integration can tap into the full knowledge of their codebase, no matter the size, with a simple chat interface. They can quickly and easily find, understand, explain, generate, refactor, and debug code right from their IDE throughout the software development lifecycle.
```
Comment (sqs) [inline content/blogposts/2023/cody-gitlab-integration:24]:
```suggestion
With Sourcegraph’s Cody, customers using the integration can tap into the full knowledge of their codebase, no matter the size, with a simple chat interface. They can quickly and easily find, understand, explain, generate, refactor, and debug code right from their IDE throughout the software development lifecycle. 
```

File: content/blogposts/2023/cody-gitlab-integration [hunk 4]
```diff
@@ -0,0 +1,36 @@
+---
+title: "Cody Now Integrates with GitLab"
+description: "GitLab customers can leverage Cody to tap into the full knowledge of their codebase, no matter the size, with a simple chat interface."
```
Comment (sqs) [inline content/blogposts/2023/cody-gitlab-integration:3]:
```suggestion
description: "GitLab customers can use Cody to tap into the full knowledge of their codebase, no matter the size, with a simple chat interface."
```

File: content/blogposts/2023/cody-gitlab-integration [hunk 5]
```diff
@@ -0,0 +1,36 @@
+---
+title: "Cody Now Integrates with GitLab"
```
Comment (sqs) [inline content/blogposts/2023/cody-gitlab-integration:2]:
```suggestion
title: "Cody now integrates with GitLab"
```

File: content/blogposts/2023/cody-gitlab-integration.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
+---
+title: "Cody Now Integrates with GitLab"
```
Comment (sqs) [inline content/blogposts/2023/cody-gitlab-integration.md:2]:
```suggestion
title: "Cody now integrates with GitLab"
```

File: content/blogposts/2023/cody-gitlab-integration.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+    url: https://twitter.com/amieroth
+publishDate: 2023-06-26T12:00
+tags: [blog]
+slug: "cody-gitlab-integration"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/Sourcegraph%20x%20Gitlab.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/Sourcegraph%20x%20Gitlab.png
+---
+
+As developers continue to lean into AI, Sourcegraph is making its AI coding assistant, Cody, more broadly available.
```
Comment (sqs) [inline content/blogposts/2023/cody-gitlab-integration.md:15]:
```suggestion
As developers continue to lean into AI, Sourcegraph is making its AI coding assistant, [Cody](https://about.sourcegraph.com/cody), more broadly available. 
```

File: content/blogposts/2023/cody-gitlab-integration.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+Today, Sourcegraph announced that Cody now integrates with GitLab, the most comprehensive AI-powered enterprise DevSecOps platform. Cody provides customers with multiple ways to take advantage of AI capabilities. 
+
+Key features of the integration include:
+* **Chat-based interface in your IDE:** Cody natively integrates with VS Code and comes pre-packaged with 9 recipes that allow devs to explain, understand, generate, translate, summarize, and detect ‘code smell’ across repositories and even generate fixes and unit tests. 
+* **Powerful and accurate AI:** Cody leverages Sourcegraph’s code graph to fetch code context, resulting in more precise outputs in line with an organization’s codebase and best practices. Developers can work more efficiently and intelligently within their editors while referencing knowledge from their entire codebase via AI.
+* **Accelerated Development:** By seamlessly connecting an organization's private codebase, documentation, and language libraries, Cody empowers developers to move even faster, delivering results quickly and reliably.
```
Comment (sqs) [inline content/blogposts/2023/cody-gitlab-integration.md:22]:
```suggestion
* **Accelerated development:** By seamlessly connecting an organization's private codebase, documentation, and language libraries, Cody empowers developers to move even faster, delivering results quickly and reliably.
```

File: content/blogposts/2023/cody-gitlab-integration.md [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+With Sourcegraph’s Cody, customers leveraging the integration can tap into the full knowledge of their codebase, no matter the size, with a simple chat interface. They can quickly and easily find, understand, explain, generate, refactor, and debug code right from their IDE throughout the software development lifecycle.
```
Comment (sqs) [inline content/blogposts/2023/cody-gitlab-integration.md:24]:
```suggestion
With Sourcegraph’s Cody, customers using the integration can tap into the full knowledge of their codebase, no matter the size, with a simple chat interface. They can quickly and easily find, understand, explain, generate, refactor, and debug code right from their IDE throughout the software development lifecycle. 
```

**Pull request merged**